### ISSUE TITLE
Create a staging branch that has the same CI actions as master

### ISSUE THREAD

**Issue by jerlich** [NeuromatchAcademy/course-content#253]
This would allow waxers to merge into staging and then see if as if it was in master but without putting it in master.

**Comment by mwaskom** [comment]
The CI actions run on feature branches when a PR is opened onto master, and merge is blocked until they complete/pass tests. We use day-specific feature branches because multiple people are pushing content in parallel.
